### PR TITLE
🤞 Fix 🤞 unicode 🤞 text 🤞 encoding. 🤞 Probably. 🤞 

### PR DESCRIPTION
**Pull request by devoxel**
wow

### CODE REVIEW

File: steely/plugins/snoop.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def snoop(text):
     r = requests.post(GIZOOGLE_URL, data={'translatetext': text})
     html = r.text.replace('height:250px;"/>', 'height:250px;">')  # Don't ask
     soup = BeautifulSoup(html, 'html.parser')
-    return soup.textarea.string
+    return soup.textarea.string.encode('utf-8')
```
Comment (byxor) [inline steely/plugins/snoop.py]: `string#encode` turns the string into a bytes object. Are you sure this is what we want to do?

**Comment by CianLR** [comment]
I've a feeling this may not work. It's easy enough to run outside of steely if you want to check

**Comment by devoxel** [comment]
worked for my testing, dunno if it works for steely though

**Review by CianLR — APPROVED** [review]
Ah, if it works as the function on its own then I'm sure steely can handle it. It's all python3 which is unicode by default or w/e

**Comment by sentriz** [comment]
i checked the branch out, its on lou now

**Comment by devoxel** [comment]
RIP

**Pull request closed without merging**